### PR TITLE
Remove PARENT_NODE from BetterStandardPrinter

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: src/PhpParser/NodeTraverser/RectorNodeTraverser.php [hunk 1]
```diff
@@ -14,7 +14,9 @@ final class RectorNodeTraverser extends NodeTraverser
 {
     private bool $areNodeVisitorsPrepared = false;
 
-    /** @var PhpRectorInterface[]|NodeVisitor[] */
+    /**
+     * @var PhpRectorInterface[]|NodeVisitor[]
+     */
```
Comment (samsonasik) [inline src/PhpParser/NodeTraverser/RectorNodeTraverser.php]: This one need to be reverted as cause another bug that I reverted at main branch, I am removing STOP_TRAVERAL on direct refactor() return on downgrade rules.

Comment (samsonasik) [reply src/PhpParser/NodeTraverser/RectorNodeTraverser.php]: I will resolve the conflict.

Comment (samsonasik) [reply src/PhpParser/NodeTraverser/RectorNodeTraverser.php]: Fixed conflict.

**Comment by TomasVotruba** [comment]
:+1:

**Pull request merged**